### PR TITLE
Update oauth docs

### PR DESCRIPTION
**Pull request by AlmightyYakob**
This PR adds documentation for setting up oauth login between the API and client.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,27 @@ but allows developers to run Python code on their native system.
...
 
+## Setup OAuth login
+### API
+1. Navigate to the http://localhost:8000/admin, logging in with your admin user if you're not already logged in
+2. Under the `DJANGO OAUTH TOOLKIT` section click the `Add` in the `Applications` row. This will bring you to the creation dialog for an (oauth) application.
+3. In the redirect uris field, enter `http://localhost:8080/`
+4. Select `Public` for the Client Type field
+5. Select `Authorization Code` for the Authorization Grant Type field
+6. Delete everything in the Client Secret field, leaving it blank
+7. Enter "Multinet GUI" or something similar for the Name field
```
Comment (waxlamp) [inline README.md:56]:
```suggestion
7. Enter "Multinet GUI" for the Name field
```

File: README.md [hunk 2]
```diff
@@ -45,6 +45,27 @@ but allows developers to run Python code on their native system.
...
+9. In the bottom right, click Save
+
+### Client
+1. If you haven't already, copy the `.env.default` file to `.env`
+2. Ensure there's a line that reads `VUE_APP_OAUTH_API_ROOT=http://localhost:8000/oauth/`
+3. Ensure there's a line that reads `VUE_APP_OAUTH_CLIENT_ID=<the client id you copied above>`
+4. Save the file
+5. Restart your local dev server
+
+After these steps, you should be able to login to the API from the client now.
```
Comment (waxlamp) [inline README.md:67]:
```suggestion
After these steps, you should be able to login to the API from the client.
```

**Review by waxlamp — APPROVED** [review]
Just a couple of suggestions; otherwise looks great.

**Review by naglepuff — COMMENTED** [review]
Looks good to me, thanks for adding this.

**Pull request merged**